mastodon/src/main/res/values/strings_mo.xml

```diff
@@ -87,7 +87,9 @@
     <string name="mo_setting_disable_swipe_summary">Swipe to change viewed timeline</string>
     <string name="mo_setting_haptic_feedback_summary">Vibrate when interacting with posts</string>
     <string name="mo_swap_bookmark_with_reblog_summary">Bookmark or reblog posts from the notification</string>
+    <string name="mo_settings_show_posts_without_alt_summary">Posts will be hidden in all timelines, but can be revealed in threads and notifications</string>
```

Comment:
Not sure about the `all`, I think just timelines would be fine by itself, but I'm fine with it being either way.

Reply:
All/Alle should be updated in the German translation aswell, if someone else insists on it.